wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -21,7 +21,7 @@
       },
       {
         "package": "DifferenceKit",
-        "repositoryURL": "https://github.com/wireapp/DifferenceKit",
+        "repositoryURL": "https://github.com/wireapp/DifferenceKit/",
```

Comment:
Commit on purpose? I think we can fix it, because I see the same change often in my branches

Reply:
Committed by mistake. It does come back as unstaged changes quite often

Reply:
reverted